src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -14,10 +14,14 @@
     <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
     <FileAlignment>512</FileAlignment>
     <DocumentationFile>$(OutputPath)System.Collections.Immutable.xml</DocumentationFile>
-    <SccProjectName>SAK</SccProjectName>
-    <SccLocalPath>SAK</SccLocalPath>
-    <SccAuxPath>SAK</SccAuxPath>
-    <SccProvider>SAK</SccProvider>
+    <SccProjectName>
+    </SccProjectName>
+    <SccLocalPath>
+    </SccLocalPath>
+    <SccAuxPath>
+    </SccAuxPath>
+    <SccProvider>
+    </SccProvider>
```

Comment:
I believe Visual Studio sets the properties to empty rather than remove them when you remove source control bindings in the IDE.

:exclamation: These properties should be removed (by editing the project file) rather than just being set to empty.